src/card/components/fields.jsx

```diff
@@ -69,14 +72,40 @@ export function CardField({ cspNonce, onChange, styleObject = {}, placeholder =
     useEffect(() => {
         autoFocusRef(numberRef);
     }, []);
+
+    useEffect(() => {
+        const { field, errors } = gqlErrorsObject;
+
+        if (field === CARD_FIELD_TYPE.NUMBER) {
+            const hasQQLErros = errors.length > 0;
```

Comment:
Is this supposed to be this?
```suggestion
            const hasGQLErrors = errors.length > 0;
```

Reply:
Thanks! I have applied a fix for this.